enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/controller/api/IdentityVerificationController.java

```diff
@@ -169,6 +174,9 @@ public ResponseEntity<Response> initializeIdentityVerification(@EncryptedRequest
         final OwnerId ownerId = PowerAuthUtil.getOwnerId(apiAuthentication);
         final String processId = request.getRequestObject().getProcessId();
 
+        // Lock onboarding process
```

Comment:
I suggest comment reasons, not what is going on. This is obvious, since the method `verifyProcessIdAndLock` is called. Or if you want to, let's log e.g. `logger.debug("Locking process ID: {}", processId);`

Reply:
Yes, I will switch to debug logging. The reason for the comments is that I wanted to make sure in each controller it is clear whether locking is on or off.

Reply:
Sure, I will use logging.